draft-irtf-panrg-path-properties.md

```diff
@@ -143,26 +146,26 @@ Therefore, a new technology may implement an existing use case related to differ
 
 ## Path Selection
 
-Entities can choose what traffic to send over which path or subset of paths.
-A node might be able to select between a set of paths, either if there are several paths to the same destination (e.g., in case of a mobile device with two wireless interfaces, both providing a path), or if there are several destinations, and thus several paths, providing the same service (e.g., Application-Layer Traffic Optimization (ALTO) {{RFC5693}}, an application layer peer-to-peer protocol allowing hosts a better-than-random peer selection).
+Nodes may be able to send flows via one out of multiple possible paths, or via a subsets of paths, and an entity may be able to select which path(s) to use.
```

Comment:
Maybe we can rephrase this to clarify that these actions are not exclusive:
"Nodes may be able to send flows via one (or a subset) out of multiple possible paths and an entity may be able to influence the path selection."

Reply:
I'm not entirely convinced it said that the actions were exclusive, but I'm fine with shortening it a bit.
I like "influence", but I didn't want to explain path selection with itself, so I went for "influence the decision which path(s) to use". Does this work?

Reply:
Yes, that sounds good